back/config/tenant_templates/base.yml

```diff
@@ -353,6 +348,21 @@ models:
     - project_ref: *open_idea_project
       topic_ref: *topic-other
       ordering: 15
+  phase:
+    - &open_idea_project_current
+      title_multiloc: phases.open_idea_phase_title
+      project_ref: *open_idea_project
+      start_at_timediff: -1440
+      end_at: null
+      participation_method: ideation
+      campaigns_settings:
+        project_phase_started: true
+      created_at: '2018-03-28 13:35:05'
```

Comment:
I don't think we need to set created_at and updated_at. On active platforms this could potentially be a bit misleading e.g. for statistics used to monitor company goals.